spec/setup_acceptance_node.pp

```diff
@@ -6,6 +6,10 @@
       ensure => installed,
     }
 
+    package { 'epel-release':
```

Comment:
Is EPEL still needed for these tests to pass?

Reply:
Yes, we need libmodulemd2 from there.